hub/saml-soap-proxy/src/main/resources/supported-msa-versions.yml

```diff
@@ -1,8 +1,6 @@
 versions:
   - 0
   - "null"
-  - 3.0.3-830
```

Comment:
Yes they do. And we don't want to support it.

As far as we know, the only thing this does is make the grafana dashboard say "Yes" or "no". 4.1 has been in-use and accidentally "unsupported" according to this file, and it doesn't seem to have had any effect, bad or otherwise :-)